app/assets/javascripts/discourse/app/templates/application.hbs

```diff
@@ -25,6 +25,8 @@
       {{create-topics-notice}}
       {{plugin-outlet name="top-notices" connectorTagName="div" args=(hash currentPath=router._router.currentPath)}}
     </div>
+
+    <Sidebar />
```

Comment:
Should the `if currentUser.experimental_sidebar_enabled` be outside the component? Happy either way just wondering